cli/command/system/prune.go

```diff
@@ -1,3 +1,7 @@
+// FIXME(thaJeztah): remove once we are a module; the go:build directive prevents go from downgrading language version to go1.16:
+//go:build go1.24
+// +build go1.24
```

Comment:
```suggestion
```

This makes linter angry

Reply:
Yeah. Made me angry as well, because my IDE tried to be smart ... sigh...